src/test_l1_dsAudio.c

```diff
@@ -894,9 +915,11 @@ void test_l1_dsAudio_negative_dsGetAudioCompression(void) {
 	int compression[NUM_OF_PORTS];
 
 	// Step 01: Attempt to get audio compression without initializing
+	#ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
add else, most of the API returns dsERR_INVALID_PARAM for the invalid handle. API checks with internally stored handles with handles passed to that function. 
Do the same for all other APIs